packages/blob/src/index.js

```diff
@@ -27,7 +30,7 @@ export function createBlobURL( file ) {
  *
  * @param {string} url The blob URL.
  *
- * @return {?File} The file for the blob URL.
+ * @return {File|undefined} The file for the blob URL.
```

Comment:
`{?File}` is a _nullable_ type in JSDoc, equivalent to `File | null`, which isn't accurate.

Reply:
> `{?File}` is a _nullable_ type in JSDoc, equivalent to `File | null`, which isn't accurate.

Yeah, I think you'll find a lot of examples like this in the codebase, where nullable type is used incorrectly as interchangeable with "optional" or "possibly undefined".